src/helpers/utility.js

```diff
@@ -226,9 +226,12 @@ export function getEventDesc(event, currentAddress) {
     case EVENT_STATUS.CREATED: return 'creating';
     case EVENT_STATUS.PRE_BETTING: return 'predictionComingSoon';
     case EVENT_STATUS.BETTING: return 'predictionInProgress';
+    case EVENT_STATUS.PRE_RESULT_SETTING:
```

Comment:
fyi you can combine multiple case statements together
```
case EVENT_STATUS.PRE_RESULT_SETTING:
case EVENT_STATUS.ORACLE_RESULT_SETTING: {
      if (centralizedOracle !== currentAddress) return 'arbitrationComingSoon';
      return 'resultSettingComingSoon';
}
```

Reply:
yeah, but  we cannot combine here, since if centralizedOracle === currentAddress and its oracle_result_setting, it should be result setting in progress

Reply:
oops sorry didnt notice the strings were different.